packages/jest-editor-support/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "jest-editor-support",
+  "version": "17.0.3",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/facebook/jest.git"
+  },
+  "license": "BSD-3-Clause",
+  "main": "build/index.js",
+  "scripts": {
+    "test": "../../packages/jest-cli/bin/jest.js"
+  },
+  "dependencies": {
+    "typescript": "^2.0.10"
```

Comment:
we chatted about this IRL, will aim to make Typescript an optional dependency, then in vscode-jest we can use both depending on the extension and in nuclide, they can use just Babel 👍